### DISCUSSION TITLE
Wallet Testing help!! help!!! help!!!

### DISCUSSION THREAD

**Discussion by Ryujueon** [WalletConnect/walletconnect-monorepo#5230] [Developer Support]
We recently submitted our wallet for approval and received the following message:

![walletconnet_1](https://github.com/user-attachments/assets/e8d72643-d10a-4c59-bb10-f45b069202ed)

"Wallet Testing - Could not verify if testing has been done on this wallet. As we do not have available V2 metric data for Sign. Please test your integration with our test wallet and submit for approval if successful. Link: https://react-web3wallet.vercel.app/"

We have diligently tested our wallet through the provided test site and have found that everything appears to be functioning normally. Our wallet supports the Avalanche C chain, and both wallet connection and signing processes were completed successfully during our testing.

![walletconnet_test_2](https://github.com/user-attachments/assets/7e13b49a-8678-4c55-b4ee-7f75e28badbe)
![walletconnet_test_1](https://github.com/user-attachments/assets/1af246b5-690d-4a39-a017-baf39c3a92d5)

![walletconnet_test_3](https://github.com/user-attachments/assets/20bf0f94-34d9-4112-a321-e71f2575301d)

We are looking for the reason for the rejection but have not been able to find it. Could you please provide more detailed information about the reasons for the rejection? This will help us to quickly address the issues and resubmit for approval.

Thank you for your help with this issue. We look forward to your guidance and support to expedite the wallet approval process.

Help!!! Help!! Help!!!

**Comment by glitch-txs** [comment]
Hello, could you share your project ID?

**Reply by Ryujueon** [reply]
Hello, 
project ID : 46bd25ad1aff45fc5b76aaa143dc4bc4

**Reply by glitch-txs** [reply]
Is this only available in Korea?

**Reply by Ryujueon** [reply]
yes, Available only in Korea.

**Reply by Ryujueon** [reply]
We are continuing to test Wallet Connect, and I even tried purchasing NFTs after connecting my wallet on Opensea. Everything worked normally. Please find the attached image for reference.

![walletconnet_test_4](https://github.com/user-attachments/assets/3ea7cbb9-7504-467b-a681-091f76231753)

Additionally, We are doing our best to get approval quickly.

Thank you for your attention to this matter.

**Reply by glitch-txs** [reply]
why is it shown as an extension in that screenshot?

Could you re-submit it please and send a video with the connection flow + signing to cloud@walletconnect.com

Thank you!

**Reply by Ryujueon** [reply]
I have re-submitted the approval request and attached a video demonstrating the connection flow and signing process. The video has been sent to cloud@walletconnect.com for your review.

I appreciate your assistance and look forward to your feedback at your earliest convenience.

Thank you very much for your support.

**Comment by Ryujueon** [comment]
Dear WalletConnect Manager,

I hope this message finds you well.
My name is Ryu Jueon, and I am the UPTN Station manager
(- Project ID : 46bd25ad1aff45fc5b76aaa143dc4bc4)
I wanted to follow up regarding the deeplink scheme (uptn://dapp-web.uptn.io/wc/) that was submitted in WalletConnect Explorer. 
We have made the necessary changes and submitted them for review, but the status is still pending.
Our AOS developers are eager to proceed with development using this scheme, but the continued pending status is causing delays.
Could you please expedite the review process? We would greatly appreciate your prompt attention to this matter.
Thank you very much for your support.
Best regards,
